### ISSUE TITLE
[MINIPLUS] [ES] Shutdown from emulationstation hangs and freezes on splash screen

### ISSUE THREAD

**Issue by efylan** [game-de-it/plumOS-GKD#3]
I noticed that sometimes, emulationstation will not shutdown properly and I will either have to shut it down holding the power button or connect a keyboard and press the handy and beautiful crtl+alt+del combo a dev put in this OS to restart the system and then shut down again.

It has happened to me when shutting down after changing the system settings on the menu or playing saturn and dreamcast games, I do not know what causes it or if any of this has anything to do with it.

**Comment by game-de-it** [comment]
You may be using a poor quality SD card.

The more capacity the better the quality of the SD card, so I use an SD card of 64GB or more for the operating system.

For the same reason, it's better to use 128GB or more for the SD2 to store ROM files.

Of course, there are compatibility issues with SD cards, so it's hard to say what's good or bad.

**Comment by game-de-it** [comment]
Okay, so what game is hanging up?

**Comment by efylan** [comment]
Sometimes mame roms, ports, fbneo, n64... it seems to be random, however I think its just the input that stops working, it stays on the game list, because I can restart the miniplus with the keyboard.

**Comment by game-de-it** [comment]
Depending on the game, the emulator will hang.

The same can be said for other CFWs.
For example, there are many Sega Saturn games that black out on ROCKNIX.

To find an emulator that works properly, you can try using the emulator on LOVELYCHILD's side, or switch emulators in the Emulationstation settings and check that they work.

If this still doesn't work, you have no choice but to give up.
Emulators are not all-purpose, and they may or may not work depending on the OS and emulator version.

**Comment by efylan** [comment]
mmm, the system becomes unresponsive to the controller _before_ it has any chance to load the emulator, restarting and trying again will make it work, _it doesn''t even go into the emulator_, no splash, no dark screen, just the unresponsive game list. Restarting makes it okay again, but it's uncomfortable to do this 2 to 3 times a day. And this does not happen on 2023 firmware, I'll keep testing on v0.1

**Comment by game-de-it** [comment]
I don't understand the situation exactly, so please let me explain.

1. Emulationstation starts
2. Select an item such as ports, fbneo, n64, etc. and press the A button
3. The ROM selection screen appears, select the desired ROM and press the A button
4. The ROM selection screen remains displayed and the controller becomes ineffective

Is this explanation correct for this situation?

**Comment by efylan** [comment]
Yes that is correct, then I proceed to shutdown, or restart with a keyboard connected, and try again and most of the timethe game will load, I don'k know if it's because I did something before opening the rom, but I will keep detailing this description if it keeps happening.
